lib/fennec/stun/auth.ex

```diff
@@ -0,0 +1,66 @@
+defmodule Fennec.STUN.Auth do
```

Comment:
Well, it's still `STUN` protocol, so I'm not sure it fits. `Fennec` is also to generic.